### PR TITLE
I recommend using the hostname for the VM name

### PR DESCRIPTION
**Pull request by rpletka**
or add a new variable for the vm name.

### CODE REVIEW

**Comment by rpletka** [comment]
Signed-off-by: Ryan Pletka rpletka@vmware.com

File: nsxt_yaml/basic_topology.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@
         datastore: "{{hostvars['localhost'].vcenter_datastore}}"
         portgroup: "{{hostvars['localhost'].mgmt_portgroup}}"
         cluster: "{{hostvars['localhost'].vcenter_cluster}}"
-        vmname: "nsxt-manager"
+        vmname: "{{hostvars['localhost'].nsx_manager_assigned_hostname}}.{{hostvars['localhost'].dns_domain}}"
```
Comment (kais66) [inline nsxt_yaml/basic_topology.yml]: How about just {{hostvars['localhost'].nsx_manager_assigned_hostname}}? I'm fine with either way.

**Comment by kais66** [comment]
> Signed-off-by: Ryan Pletka [rpletka@vmware.com](mailto:rpletka@vmware.com)

signed-off-by needs to be added to commit message according to the comment above

File: nsxt_yaml/basic_topology.yml (hunk 1 above)
Comment (rpletka) [reply nsxt_yaml/basic_topology.yml]: that sounds good.

**Pull request closed without merging**